user-guide/Advanced_Functionality/Databases/STaaS/STaaS.md

```diff
@@ -108,6 +108,9 @@ To **migrate existing data** to STaaS, the following limitations apply:
 
 In addition, the following **other limitations** currently apply:
 
+- TTL is only supported for trending datatypes. The TTL values are not configurable and changing the values in CUBE will not have any effect.
```

Comment:
this is not totally true... changing these values still affect how much data is returned, where the amount of data is the minimum value of the TTL configured in cube and our default ttl